dev-tools/packaging/settings.go

```diff
@@ -110,6 +110,65 @@ func (proj BinarySpec) GetPackageName(version string, platform string) string {
 	return buf.String()
 }
 
+func (proj BinarySpec) GetRootDir(version string, platform string) string {
```

Comment:
nit: I would add a godoc here saying what this Func is aiming to do

Reply:
added some docs in 39e20b49a1b083ad36b631e4a442c95aa6a559e9